documentation/airport-guides/katl.md

```diff
@@ -0,0 +1,107 @@
+### About this facility
+Atlanta TRACON (A80), Peachtree City, Georgia, USA  
+1,193,710 annual operations in 2017 ([FAA ATADS](https://aspm.faa.gov/opsnet/sys/Tracon.asp))
+
+Located 20 miles southwest of KATL and almost 30 miles from Downtown Atlanta, The A80 TRACON is one of the busiest control facilities in the world. It controls a huge area over north central
+Georgia, as well as portions of South Carolina and Alabama, and it covers over 200 airports, including the world's busiest airport, Hartsfield-Jackson Atlanta Int'l (KATL).
```

Comment:
remove new line and add a dash so it’s ‘north-central’

‘It controls a huge area over north-central Georgia, as well as portions of South Carolina and Alabama, covering 200 airports, including the world’s busiest airport, Hartsfield–Jackson Atlanta International (KATL).

replace the paragraph with that ^